swaglyrics/cli.py

```diff
@@ -33,6 +34,8 @@ def stripper(song: str, artist: str) -> str:
 	:return: formatted url path
 	"""
 	song = re.sub(brc, '', song).strip()  # remove braces and included text with feat and text after '- '
+	if not re.sub(nlt, '', song):  # check if title contains strictly non-latin characters
```

Comment:
You're not checking for artist, in any case, I don't think the stripper function needs modification. You should be only modifying get_lyrics for this one.

Reply:
Ohhhhh, I thought the task was strictly for non-Latin titles. Makes sense, I'll go fix it

Reply:
Oh haha, titles included both song and artist :)